RawWaterManagement/OpenChannelSystem/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,94 @@
+      "coordinates": [
+        -82.1141235,
+        -159.124903
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Young military guy nearly dinner. Course fear certainly reveal while truth. Thousand reduce ready.",
+      "addressLocality": "Federal or we business. Peace body bank seem idea.",
+      "addressRegion": "Impact national respond as like up significant. Officer wonder inside worry property response himself choice.",
+      "addressCountry": "Lead artist summer site newspaper century land. Course very tough business. Foot full blood stuff behind.",
+      "postalCode": "Service easy sure best throw. Budget experience run past go sister. Table thing best.",
+      "postOfficeBoxNumber": "Be candidate buy traditional air word herself product. High share parent stock almost. Success poor source might.",
+      "areaServed": "Both example finally fish year evening power local. Control fire management left. Almost her ground choose condition say player. Go my model stock let prepare."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Cup bar series assume majority defense that by. Many manager somebody approach. Lay leg page summer line."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:NXDF:16687280",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2003-07-15T09:19:21Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1992-11-16T21:11:30Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Increase film these such cover. Analysis total woman or official successful money."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Lot section establish most. Movie important white himself somebody. Within buy once land energy avoid."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Space into politics rich compare international certainly. Television its place just let."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Leave finally may last mission. Day by laugh test former. Artist sort process shake lead when year central. Age until democratic major."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Three course understand. White week catch one."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:MKXR:87751348",
+      "urn:ngsi-ld:dataModel:items:BDXZ:05485215"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:IYBR:25722869",
+      "urn:ngsi-ld:dataModel:items:FAYI:83539269"
+    ]
+  },
+  "type": "OpenChannelSystem",
+  "isComposedOf": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:isComposedOf:VXIQ:86814542"
+  },
+  "hasSubSystem": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:hasSubSystem:DBFG:88836989"
+  },
+  "mostUpstreamPoint": {
```

Comment:
type should be Relationship
The proper way to model Relationships is as follow :

"observedBy":{
	"type":"Relationship",
	"object":" urn:ngsi-ld:Device:01 "	}